### DISCUSSION TITLE
where can I find all the achievement badges repo?

### DISCUSSION THREAD

**Discussion by moesi88** [community/community#19354] [Other Feature Feedback, Questions, & Ideas] [answered]
and does it explain the ranks of each?

**Comment by Maher-Reven** [comment, marked as answer]
https://github.com/Schweinepriester/github-profile-achievements

you can find it here @moesi88

**Comment by greybax** [comment]
https://github.com/Schweinepriester/github-profile-achievements seems it's not like an official repo
